Favor/Favor/Sources/Scenes/AnniversaryList/Reactors/AnniversaryListViewReactor.swift

```diff
@@ -16,56 +16,68 @@ import RxFlow
 final class AnniversaryListViewReactor: BaseAnniversaryListViewReactor, Reactor, Stepper {
   typealias Section = AnniversaryListSection
   typealias Item = AnniversaryListSectionItem
-
+  
   // MARK: - Properties
-
+  
   var initialState: State
   var steps = PublishRelay<Step>()
-  let networking = AnniversaryNetworking()
-
+  let workbench = RealmWorkbench()
+  let anniversaryListType: AnniversaryListType
+  
   enum Action {
     case viewNeedsLoaded
     case editButtonDidTap
     case pinButtonDidTap(Anniversary)
   }
-
+  
   enum Mutation {
     case updateAnniversaries([Anniversary])
-    case updatePinnedSection([Item])
     case updateAllSection([Item])
   }
-
+  
   struct State {
     var anniversaries: [Anniversary] = []
     var sections: [Section] = []
-    var items: [[Item]] = []
-    var pinnedItems: [Item] = []
-    var allItems: [Item] = []
+    var items: [Item] = []
   }
-
+  
   // MARK: - Initializer
-
-  override init() {
+  
+  init(anniversaryListType: AnniversaryListType) {
+    self.anniversaryListType = anniversaryListType
     self.initialState = State()
     super.init()
+    
+    if case AnniversaryListType.friend(let friend) = anniversaryListType {
+      self.setupFriendFetcher(with: friend)
+    }
   }
-
+  
   // MARK: - Functions
-
+  
   func mutate(action: Action) -> Observable<Mutation> {
     switch action {
     case .viewNeedsLoaded:
-      return self.userFetcher.fetch()
-        .flatMap { (_, user) -> Observable<Mutation> in
-          guard let user = user.first else { return .empty() }
-          let anniversaries = user.anniversaryList
-          return .just(.updateAnniversaries(anniversaries))
-        }
-
+      switch self.anniversaryListType {
+      case .mine:
+        return self.userFetcher.fetch()
+          .flatMap { (_, user) -> Observable<Mutation> in
+            guard let user = user.first else { return .empty() }
+            let anniversaries = user.anniversaryList
+            return .just(.updateAnniversaries(anniversaries))
+          }
+      case .friend:
+        return self.friendFetcher.fetch()
+          .flatMap { (status, friend) -> Observable<Mutation> in
+            guard let friend = friend.first else { return .empty() }
+            return .just(.updateAnniversaries(friend.anniversaryList.sort()))
```

Comment:
밑의 `transform`에서 공통적으로 `sort` 처리 되는 것 같아서 여기서는 정렬 처리 없이 넘겨도 될 듯..?
```suggestion
            return .just(.updateAnniversaries(friend.anniversaryList))
```